### PR TITLE
Adding support for AWS EventBridge

### PR DESCRIPTION
**Pull request by ekifel**
Issue #387 

```
  NAME                                          MONTHLY QTY  UNIT       PRICE   HOURLY COST  MONTHLY COST  

  aws_cloudwatch_event_bus.my_events                                                                       
  ├─ Custom events published                              1  1M events  1.0000       0.0014        1.0000  
  ├─ Third-party events published                         2  1M events  1.0000       0.0027        2.0000  
  ├─ Archive processing                                 100  GB         0.1000       0.0137       10.0000  
  ├─ Archive storage                                    200  GB-months  0.0230       0.0063        4.6000  
  └─ Schema discovery                                     1  1M events  0.1000       0.0001        0.1000  
  Total                                                                              0.0242       17.7000  
```

### CODE REVIEW

File: internal/providers/terraform/aws/cloudwatch_event_bus.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+				Unit:            "events",
+				UnitMultiplier:  1000000,
+				MonthlyQuantity: monthlyIngestedEvents,
+				ProductFilter: &schema.ProductFilter{
+					VendorName:    strPtr("aws"),
+					Region:        strPtr(region),
+					Service:       strPtr("AWSEvents"),
+					ProductFamily: strPtr("EventBridge"),
+					AttributeFilters: []*schema.AttributeFilter{
+						{Key: "eventType", Value: strPtr("Discovery Event")},
```
Comment (alikhajeh1) [inline internal/providers/terraform/aws/cloudwatch_event_bus.go:83]:
```suggestion
 						{Key: "usagetype", ValueRegex: strPtr("/Event-8K-Chunks/")},
```
We try to use the usagetype filter where possible (we have some ideas for the future that could use that)

File: internal/providers/terraform/aws/cloudwatch_event_bus.go [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+				Unit:            "events",
+				UnitMultiplier:  1000000,
+				MonthlyQuantity: monthlyPartnerEvents,
+				ProductFilter: &schema.ProductFilter{
+					VendorName:    strPtr("aws"),
+					Region:        strPtr(region),
+					Service:       strPtr("AWSEvents"),
+					ProductFamily: strPtr("EventBridge"),
+					AttributeFilters: []*schema.AttributeFilter{
+						{Key: "eventType", Value: strPtr("Partner Event")},
```
Comment (alikhajeh1) [inline internal/providers/terraform/aws/cloudwatch_event_bus.go:68]:
```suggestion
						{Key: "eventType", Value: strPtr("Partner Event")},
						{Key: "usagetype", ValueRegex: strPtr("/Event-64K-Chunks/")},
```
This ensures that in the future if they add other usagetypes for the Partner Event, we'll still find the right one

File: internal/providers/terraform/aws/cloudwatch_event_bus.go [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+				Unit:            "events",
+				UnitMultiplier:  1000000,
+				MonthlyQuantity: monthlyCustomEvents,
+				ProductFilter: &schema.ProductFilter{
+					VendorName:    strPtr("aws"),
+					Region:        strPtr(region),
+					Service:       strPtr("AWSEvents"),
+					ProductFamily: strPtr("EventBridge"),
+					AttributeFilters: []*schema.AttributeFilter{
+						{Key: "eventType", Value: strPtr("Custom Event")},
```
Comment (alikhajeh1) [inline internal/providers/terraform/aws/cloudwatch_event_bus.go:53]:
```suggestion
						{Key: "eventType", Value: strPtr("Custom Event")},
						{Key: "usagetype", ValueRegex: strPtr("/Event-64K-Chunks/")},
```
This ensures that in the future if they add other usagetypes for the Custom Event, we'll still find the right one

**Review by alikhajeh1 — CHANGES_REQUESTED** [review]
Thanks @ekifel - this is great, it's almost there. My suggestions were mostly related to the cost component names/units so plz see that comment first, the rest of the code will need some tweaks based on that. I also left a few tweaks to the filters.

**Comment by ekifel** [comment]
@alikhajeh1 this is ready for review/merge

**Pull request merged**